### ISSUE TITLE
Can't comment on members only chat

### ISSUE THREAD

**Issue by tv1mdb** [LiveTL/LiveTL#91]
**Describe the bug**
Can't comment on members only chat even tho I'm a member

**To Reproduce**
1. Go to video with members only chat
2. Click on liveTL
3. "Members-only mode is on" appears and can't comment (I can by using the pop-up chat option)

**Expected behavior**
https://i.imgur.com/yaqZKry.png

**Screenshots**
https://i.imgur.com/YGSMg2f.png

**Desktop (please complete the following information):**
 - OS: w10
 - Browser Brave
 - whatever the newest is

Edit:
Tried to delete all cookies but changed nothing

**Comment by KentoNishi** [comment]
I think this just means members only mode is on for the stream, meaning the streamer chose to only allow comments from members

**Comment by KentoNishi** [comment]
Nvm ur a member, that's odd

**Comment by tv1mdb** [comment]
Update: Had some time today so I decided to watch a pekora stream and now I'm prompted to "create a channel" to be able to comment (screen below), which I already have (even have videos uploaded). Tried first to delete all cookies/cache, then to open the stream/livetl on a private windows with all other extensions disabled and still same issue.
Now tried to use a different browser (ff) and no longer have the problem.
Installed brave nightly to simulate a fresh install and I don't have this issue.
At this point I'm pointing to something being cached that does not get deleted being bugged, finding out what now is my next step.
